library/core/src/ptr/non_null.rs

```diff
@@ -1,13 +1,13 @@
 use crate::cmp::Ordering;
 use crate::marker::Unsize;
-use crate::mem::{MaybeUninit, SizedTypeProperties};
+use crate::mem::{self, MaybeUninit, SizedTypeProperties};
```

Comment:
Remove this import, we could use core::mem::* if I remember

Reply:
removed